frontend/src/components/Metrics/ReportStep/index.tsx

```diff
@@ -82,14 +86,19 @@ const ReportStep = ({ notification, handleSave }: ReportStepProps) => {
         clearInterval(timer);
       };
     }
-  }, [exportValidityTimeMin]);
+  }, [exportValidityTimeMin, isAllMetricsReady]);
+
+  useLayoutEffect(() => {
+    resetProps();
+  }, [pageType]);
```

Comment:
React Hook useLayoutEffect has a missing dependency: 'resetProps'. Either include it or remove the dependency array.

Reply:
done